R/data.R

```diff
@@ -69,17 +69,17 @@
 #'  \item{percasian}{Percent Asian.}
 #'  \item{percother}{Percent other races.}
 #'  \item{popadults}{Number of adults.}
-#'  \item{perchsd}{}
+#'  \item{perchsd}{Percent with high school diploma.}
```

Comment:
Not very reliable, but I found [someone's homework](https://www.coursehero.com/file/26646690/JohnLonaipdf/) that lists `perchsd` as "the percentage of people with a high schooldiploma in each county".

I couldn't verify this, but given the surrounding other variables on education status (`percollege` and `percprof`), I figured this to be a high school diploma.